osu.Game/Screens/Edit/Compose/ComposeScreen.cs

```diff
@@ -138,7 +137,7 @@ public override void Copy()
             // regardless of whether anything was even selected at all.
             // UX-wise this is generally strange and unexpected, but make it work anyways to preserve muscle memory.
             // note that this means that `getTimestamp()` must handle no-selection case, too.
-            hostClipboard.SetText(getTimestamp());
+            game?.CopyToClipboard(getTimestamp());
```

Comment:
I dunno that I agree with this one in particular. Anywhere else indicating that a copy happened is probably fine, but I can already *feel* the mapper power users going "this toast showing every time I ctrl-c is annoying and obscuring the playfield make it stop or at least make it a toggle".

In an environment where copying is heavily utilized as a part of a standard workflow to do things I am not sure that spamming toasts is helpful anymore.